Frontend/src/pages/AboutPage.js

```diff
@@ -15,7 +19,7 @@ function AboutPage() {
       for further analysis. This type of searchable rich data set will save researchers
       countless hours of computation as well as provide a means to identify previously
       unknown patterns and connections</p>
-      <img src="./homeImage.png" alt="image" />
```

Comment:
keep the alt here it causes an extra error and is good practice for accessibility

Reply:
It throws a linting warning because it says the alt is already implicitly included